app/views/results/_defendant_appeals.html.haml

```diff
@@ -0,0 +1,27 @@
+%table.govuk-table
```

Comment:
Most of this is copy-pasted from an existing view. Would it make sense to just have one view, and use the feature flag to toggle whether we display the final `th`/`td`? Just to avoid duplication?

Reply:
I think its easier and cleaner this way for now once we remove the faeture flag this can be removed.